tempesta_fw/cfg.c

```diff
@@ -748,28 +746,46 @@ spec_start_handling(TfwCfgSpec specs[])
 		BUG_ON(!spec->handler);
 		if (spec->handler == &tfw_cfg_handle_children)
 			BUG_ON(!spec->cleanup);
+		BUG_ON(parent && spec->allow_reconfig && !parent->allow_reconfig);
```

Comment:
Is it absolutely obvious, that the system is going to crash and we have to crash right now? Basically it's discouraged to use `BUG_ON()` construction. Can we use `WARN_ON()` instead, abort the configuration and Tempesta start procedure?